_data/master/navbars/getting-started.yml

```diff
@@ -68,6 +68,8 @@ toc:
   section:
     - title: Installation
       path: /getting-started/openshift/installation
+    - title: BYO Etcd
```

Comment:
I would suggest a title more like "Setting up a custom etcd (optional)"

I would avoid "BYO". Many people may not know what it means.